src/mappings/stateChannel.ts

```diff
@@ -79,7 +79,7 @@ export async function handleChannelFund(
   const { channelId, total } = event.args;
   const sc = await StateChannel.get(channelId.toHexString());
   assert(sc, `Expected StateChannel (${channelId.toHexString()}) to exist`);
-  sc.total = total.toBigInt();
+  sc.total = sc.total + total.toBigInt();
```

Comment:
The `total` value is not the `delta` value in the contract, do you mean the contract also need to make the change? cc @nz-neooo 
```
// transfer the balance to contract
IERC20(settings.getSQToken()).safeTransferFrom(consumer, address(this), amount);
channels[channelId].total += amount;
emit ChannelFund(channelId, channels[channelId].total);
```

Reply:
I got it wrong, will close the PR for now.